sources/packages/web/src/views/institution/Login.vue

```diff
@@ -7,37 +7,25 @@
             Welcome to StudentAid BC
           </h1>
           <p class="mb-5">
-            Login or sign up here to manage your institution account.
+            Manage your institution account using your Business or Basic BCeID
+            User ID. Learn how to setup and access your account
+            <a
+              rel="noopener"
+              target="_blank"
```

Comment:
can you add class="primary-color" for the brand blue color, i believe else it will be default  href color